SleepEngine/Engine/Jobs/Event.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+BEGIN_NAMESPACE_SLEEP
+
+class Event
+{
+    using LockGuard = std::lock_guard<std::mutex>;
+public:
+    Event() = default;
+    Event(Event&&) = default;

```

Comment:
Should you write these 2 lines?
Both constructor and move constructor will be defaulted by default.

Reply:
So yeah, they will be deleted if another ctor is added